src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationusestyle/InputAnnotationsUseStyleParams.java

```diff
@@ -1,4 +1,4 @@
-package com.puppycrawl.tools.checkstyle.checks.annotation;
+package com.puppycrawl.tools.checkstyle.checks.annotation.annotationusestyle;
```

Comment:
This class should start with `InputAnnotationUseStyle...`. It has an `s` after `Annotation`.